README.md

```diff
@@ -3,7 +3,11 @@
 # TO DO LIST STRUCTURE
 
 ## PROJECT DESCRIPTION
-This project is about building a simple to-do list project.
+This project is about building a simple to-do list project. It contains list of activities 
+needed to be carried out.
+
+- Implemented a function for the "Clear all completed" button.
+- Stored updated array items in local storage 
```

Comment:
- [ ] [Optional] In your `README.md` file, kindly do the following:

1.  Include instructions for setting up this repository on the local machine. For example, instructions to clone the repository, instructions to install all dependencies (`npm install`), instructions to start the server locally (`npm run start)` 
2. Kindly include your social media handles in the author section of this `README.md` file. This would make your `README.md` file professional and make it easy for recruiters to view your profile. 👍